File: packages/react-native-nitro-image/ios/HybridChild.swift
//
//  HybridChild.swift
//  react-native-nitro-image
//
//  Created by Marc Rousavy on 03.10.24.
//

import Foundation

class HybridChild : HybridChildSpec {
  var hybridContext = margelo.nitro.HybridContext()
  var memorySize: Int {
    return getSizeOf(self)
  }
  
  var baseValue: Double {
    return 20
  }
  var childValue: Double {
    return 30
  }
}

File: packages/react-native-nitro-test/android/src/main/java/com/margelo/nitro/image/HybridChild.kt
package com.margelo.nitro.image

import androidx.annotation.Keep
import com.facebook.proguard.annotations.DoNotStrip

@Keep
@DoNotStrip
class HybridChild: HybridChildSpec() {
    override val baseValue: Double
        get() = 20.0
    override val childValue: Double
        get() = 30.0
}
